polygon/rest/aggs.py

```diff
@@ -60,7 +60,7 @@ def get_grouped_daily_aggs(
         raw: bool = False,
         market_type: str = "stocks",
         include_otc: bool = False,
-    ) -> Union[GroupedDailyAgg, HTTPResponse]:
+    ) -> Union[Iterator[GroupedDailyAgg], HTTPResponse]:
```

Comment:
```suggestion
    ) -> Union[List[GroupedDailyAgg], HTTPResponse]:
```

`List` is what we define for [`get_aggs`](https://github.com/polygon-io/client-python/blob/2bbb5c7f8a5d52641ec688fdf560ca3785dbb5a4/polygon/rest/aggs.py#L22) so we should be consistent with that.

Reply:
👍